react-frontend/src/oldIndex.js

```diff
@@ -28,136 +28,136 @@ const AppLayout = () => {
-        element: (
-          <PrivateRoute
-            isAuthenticated={isAuthenticated}
-            element={() => (
-              <Fragment>
-                <Navbar /> <Settings />{" "}
-              </Fragment>
-            )}
-          />
-        )
-      },
-      {
-        //Will we have a signout page?
-        path: "signout",
-        element: <LoginPage />
-      },
-      {
-        path: "login",
-        element: <LoginPage />
-      },
-      {
-        path: "signup",
-        element: <SignUpPage />
-      },
-      {
-        path: "forgot",
-        element: <ForgotPasswordPage />
-      },
-      {
-        path: "*",
-        element: <ErrorPage />
-      }
-    ]
-  }
-]);
-return (
-  <>
-  {/* <Navbar /> */}
-  <Outlet />
-</>
-);
+  const router = createBrowserRouter([
+    {
+      element: <AppLayout />,
+      children: [
+        {
+          //element - is the component page. If changed to MyApp page will process.
+          path: "/",
+          element: (
+            <PrivateRoute
+              isAuthenticated={isAuthenticated}
+              element={() => (
+                <Fragment>
+                  <Navbar /> <Dashboard />{" "}
+                </Fragment>
+              )}
+            />
+          )
+        },
+        {
+          path: "dashboard",
+          element: (
+            <PrivateRoute
+              isAuthenticated={isAuthenticated}
+              element={() => (
+                <Fragment>
+                  <Navbar /> <Dashboard />{" "}
+                </Fragment>
+              )}
+            />
+          )
+        },
+        {
+          path: "shop",
+          element: (
+            <PrivateRoute
+              isAuthenticated={isAuthenticated}
+              element={() => (
+                <Fragment>
+                  <Navbar /> <Shop />{" "}
+                </Fragment>
+              )}
+            />
+          )
+        },
+        {
+          path: "tasks",
+          element: (
+            <PrivateRoute
+              isAuthenticated={isAuthenticated}
+              element={() => (
+                <Fragment>
+                  <Navbar /> <Tasks />{" "}
+                </Fragment>
+              )}
+            />
+          )
+        },
+        {
+          path: "pet",
+          element: (
+            <PrivateRoute
+              isAuthenticated={isAuthenticated}
+              element={() => (
+                <Fragment>
+                  <Navbar /> <Pet />{" "}
+                </Fragment>
+              )}
+            />
+          )
+        },
+        {
+          path: "settings",
+          element: (
+            <PrivateRoute
+              isAuthenticated={isAuthenticated}
+              element={() => (
+                <Fragment>
+                  <Navbar /> <Settings />{" "}
+                </Fragment>
+              )}
+            />
+          )
+        },
+        {
+          //Will we have a signout page?
```

Comment:
Thanks for bringing this up! I decided to have our root as the login page so that if users aren't logged in, they'll just be routed there. The other routes are helpful in navigating the login/signup/forgot password pages because they link to each other. The oldIndex.js was what we previously had and accidentally left it there while I was referencing it. Thank you for catching that! Let me know if you have any other feedback.

Reply:
Ok sounds good!